demisto_sdk/commands/format/update_generic_json.py

```diff
@@ -93,18 +93,24 @@ def remove_null_fields(self):
     def update_id(self, field='name') -> None:
         """Updates the id to be the same as the provided field ."""
         updated_integration_id_dict = {}
+        should_update_id = True
+        if self.old_file:
+            current_id = self.data.get('id')
+            old_id = self.old_file.get('id')
+            should_update_id = (current_id != old_id)
```

Comment:
In the case where the current id is different from the existing id (meaning the developer has edited the id), you would not want to overwrite it as you do in lines 109-110.

Reply:
lets discuss